content/brazilian-portuguese/arduino/_index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Arduino: Jogo de luz"
+description: "Construa um microcontrolador Arduino"
+date: 2020-07-27T14:04:59.61-07:00
+difficulty: "Intermediário"
+icon: "fas fa-microchip"
+draft: false
+---
+
+## Introdução - Construindo com Arduino
+
+Eletrônicos como microcontroladores, sensores e todos os tipos de circuitos estão ao nosso redor! Alguns são muito básicos e fornecem informações de telemetria para sistemas de tráfego. Outros executam os semáforos programados com temporização embutida.
+
+Hoje trabalharemos em algumas atividades para mostrar algumas das capacidades desta plataforma. Começaremos pequenos e simples e construiremos para maiores e mais complexos. Você aprenderá todas as informações básicas sobre como usar a placa controladora Arduino, sensores e componentes. Depois de fazer cada atividade, sinta-se à vontade para mudar algumas coisas para ver o que é diferente e obter uma compreensão mais profunda
+
+![Picture of an Arduino](./img/Arduino.png)
+
+## Os Componentes
+
+ Os componentes que usaremos para o projeto são mostrados abaixo. Certifique-se de ter todos esses componentes antes de começar:
+
+![Alt Text: Picture of Elegoo UNO Starter Kit](img/Elegoo-starter-kit.png)
+
+`Elegoo` O Kit Super Iniciante de Projeto UNO com Tutorial para Arduino inclui todo o hardware (com exceção do computador)
```

Comment:
"`Elegoo` O Kit Super Iniciante de Projeto UNO" is a product, I suggest to not translate it. Suggestion: "`Elegoo` UNO Project Super Starter Kit"